Assets/HoloToolkit/License.txt

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 Microsoft Corporation
+Copyright (c) 2017 Microsoft Corporation
```

Comment:
Should we also add the toolkit version here?

Reply:
Hmm yeah, either here or a separate version.txt. It probably makes sense to have it all-in-one.

Reply:
MRTK_Version.txt or something akin.

Reply:
I added MRTKVersion.txt to the HoloToolkit folder.